packages/apiserver/src/data-transfer/data-transfer.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@nestjs/common';
+import { Material } from '../resources/materials/material.entity';
+import { Like } from 'typeorm';
+import { deleteLocalFile, readLocalFile } from '@mimir/helper-functions';
+import { DigitalOceanService } from '@mimir/api-util';
+import * as process from 'process';
+
+@Injectable()
+export class DataTransferService {
+  constructor(private digitalOceanService: DigitalOceanService) {}
+
+  async move(): Promise<{ successes: number; errors: number }> {
```

Comment:
I would create an interface for this type `{ successes: number; errors: number }`